x-pack/plugins/security_solution/server/lib/detection_engine/routes/rules/add_prepackaged_rules_route.ts

```diff
@@ -105,6 +106,7 @@ export const createPrepackagedRules = async (
   alertsClient: AlertsClient,
   frameworkRequest: FrameworkRequest,
   maxTimelineImportExportSize: number,
+  config: ConfigType,
```

Comment:
If you push the individual types down rather than the whole configuration you should be able to do this for the types:

```ts
maxTimelineImportExportSize: ConfigType['maxTimelineImportExportSize'],
prebuiltRulesFromFileSystem: ConfigType['prebuiltRulesFromFileSystem'],
prebuiltRulesFromSavedObjects: ConfigType['prebuiltRulesFromSavedObjects']
```

The `maxTimelineImportExportSize` should really be this type rather than doing a re-typing of what the config type is. If you could add it as well that would be bonus for us 👍  and much appreciated.

Reply:
fixed in 8f9eb8247e42362f5b6328eb1c567100c179a864